src/components/Profile.tsx

```diff
@@ -58,13 +58,13 @@ function Profile() {
   const [nickname, setNickname] = useState('');
   const [location, setLocation] = useState('Baton Rouge, LA');
   const [myMood, setMood] = useState("I'm Feeling Great!");
-  const [age, setAge] = useState('18-99+');
+  const [age, setAge] = useState('Add age here');
```

Comment:
I like the default here being clear it hasn't been set yet. Should we do that for location, mood, and phone number as well?

Reply:
Yes, I believe I want to do that for all of them. I'm updating it now.

Reply:
Nice!